apps/client/src/pages/builder/sidebars/left/sections/picture/section.tsx

```
@@ -22,20 +30,199 @@ import { PictureOptions } from "./options";
+    cropHeight = Math.max(1, Math.min(cropHeight, naturalHeight - cropY));
+
+    // Ensure crop coordinates are within image bounds
+    if (
+      cropX < 0 ||
+      cropY < 0 ||
+      cropX + cropWidth > naturalWidth ||
+      cropY + cropHeight > naturalHeight
+    ) {
+      <Toast variant="error">
+        Crop coordinates are invalid. Please adjust the crop area and try again.
+      </Toast>      
```

ESLint

[error] String not marked for translation. Wrap it with <Trans>.

(lingui/no-unlocalized-strings)